src/typegenPrinter.ts

```diff
@@ -141,6 +144,9 @@ export class TypegenPrinter {
         `  allNamedTypes: NexusGenTypes['allInputTypes'] | NexusGenTypes['allOutputTypes']`,
         `  abstractTypes: NexusGenTypes['interfaceNames'] | NexusGenTypes['unionNames'];`,
         `  abstractResolveReturn: NexusGenAbstractResolveReturnTypes;`,
+        `  isTypeOfObjectNames: NexusGenIsTypeOfObjectNames;`,
+        `  resolveTypeImplemented: NexusGenResolveTypeImplemented;`,
+        `  features: NexusGenFeaturesConfig;`,
```

Comment:
Happy to get suggestions for better namings...

Reply:
- `objectsUsingAbstractStrategyIsTypeOf`
- `objectsUsingAbstractStrategyResolveType`

Reply:
`objectsUsingAbstractStrategyResolveType` -> `abstractsUsingStrategyResolveType`

Reply:
Did this